src/esss_fix_format/cli.py

```diff
@@ -42,15 +43,26 @@ def is_cpp(filename):
     return any(fnmatch(os.path.basename(filename), p) for p in CPP_PATTERNS)
 
 
-def should_format(filename):
+def should_format(filename: str, include_patterns: Iterable[str], exclude_patterns: Iterable[str]) -> Tuple[bool, str]:
     """
-    Return a tuple (fmt, reason) where fmt is True if the filename
-    is of a type that is supported by this tool.
+    Return a tuple (fmt, reason) where fmt is True if the filename should be formatted.
+
+    :param filename: file name to verify if should be formatted or not
+
+    :param include_patterns: list of file patterns to be excluded from formatting
```

Comment:
```
    :param include_patterns: list of file patterns to be excluded from formatting
    :param exclude_patterns: list of file patterns to be excluded from formatting
```
Same documentation for `include_patterns` and `exclude_patterns`, please fix `include_patterns`.

Also from the docstring is not clear the precedence between `include_patterns` and `exclude_patterns`. Could you document what happens if both patterns match the same file?

Reply:
done